tensorflow/core/kernels/dml_addn_op.cc

```diff
@@ -103,9 +94,7 @@ class DmlAddNKernel : public DmlKernel {
 // TODO(b/25387198): A special kernel exists for int32 (see aggregate_ops.cc).
 TF_CALL_float(REGISTER_KERNEL);
 TF_CALL_half(REGISTER_KERNEL);
```

Comment:
I assume uint32/uint64 were registered by mistake earlier: I don't see them registered for GPU device. Curious if this caused any problems, though?

Reply:
No problems that I know of, but TF is very careful with what types they register for GPU devices since it can affect operator placement and performance. It's better to just follow whatever CUDA registers until we have a reason to register more types. And supporting uint64 means having one more hack (aka double strides and zero the output), which I don't really want to keep if we don't have to.